tests/kernel/fatal/src/main.c

```diff
@@ -169,7 +169,9 @@ void stack_sentinel_timer(void)
 {
 	/* We need to guarantee that we receive an interrupt, so set a
 	 * k_timer and spin until we die.  Spinning alone won't work
-	 * on a tickless kernel.
```

Comment:
The functional change originally in this has been superseded by #19236.  The commit should be dropped from the PR.